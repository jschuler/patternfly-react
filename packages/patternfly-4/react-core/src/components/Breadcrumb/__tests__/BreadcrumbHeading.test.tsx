// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { BreadcrumbHeading } from '../BreadcrumbHeading';

describe('BreadcrumbHeading test', () => {
  it('BreadcrumbHeading should match snapshot', () => {
    const view = shallow(
      <BreadcrumbHeading
        children={<>ReactNode</>}
				className={"''"}
				to={"null"}
				target={"null"}
				component={<>ReactNode</>}
      />);
    expect(view).toMatchSnapshot();
  });
});
