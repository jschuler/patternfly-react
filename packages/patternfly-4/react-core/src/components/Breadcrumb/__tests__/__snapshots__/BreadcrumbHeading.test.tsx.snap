// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadcrumbHeading test BreadcrumbHeading should match snapshot 1`] = `
<li
  className="pf-c-breadcrumb__item ''"
>
  <h1
    className="pf-c-breadcrumb__heading"
  >
    <[object Object]
      aria-current="page"
      className="pf-c-breadcrumb__link pf-m-current"
      href="null"
      target="null"
    >
      ReactNode
    </[object Object]>
  </h1>
</li>
`;
